Studio/src/Application/Data/Session.cpp

```diff
@@ -684,6 +698,11 @@ double Session::update_auto_glyph_size()
   }
 
   this->auto_glyph_size_ = max_range / std::sqrt(static_cast<double>(num_particles)) / 2;
+
+  std::cerr << "max_range = " << max_range << "\n";
```

Comment:
Need to print these? I sometime keep them commented since they can be so useful when debugging (see Mesh::toImage and Python binding of Image.init).

Reply:
Yeah, I need to push a cleanup commit I already made.